src/components/Stories/index.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-console */
+import { Grid, useMediaQuery } from "@material-ui/core";
+import { useTheme } from "@material-ui/core/styles";
+import { chunk } from "lodash";
+import PropTypes from "prop-types";
+import React from "react";
+import Carousel from "react-multi-carousel";
+
+import CarouselItem from "./CarouselItem";
+import useStyles from "./useStyles";
+
+import FeaturedStoryCard from "@/pesayetu/components/FeaturedStoryCard";
+import Section from "@/pesayetu/components/Section";
+
+import "react-multi-carousel/lib/styles.css";
+
+const responsive = {
+  desktop: {
+    breakpoint: {
+      max: 3000,
+      min: 1280,
+    },
+    items: 1,
+  },
+  tablet: {
+    breakpoint: { max: 1279, min: 768 },
+    items: 1,
+  },
+  mobile: {
+    breakpoint: { max: 767, min: 0 },
+    items: 1,
+  },
+};
+
+function Stories({ featuredStoryProps, items, ...props }) {
+  const classes = useStyles(props);
+  const theme = useTheme();
+  const isTablet = useMediaQuery(theme.breakpoints.up("md"));
+  const itemsToShow = isTablet ? 6 : 3;
+  const carouselItems = chunk(items, itemsToShow);
+
+  return (
+    <Grid container direction="column">
+      <Section classes={{ root: classes.section }}>
+        <Grid item>
+          <FeaturedStoryCard {...featuredStoryProps} />
+        </Grid>
+        <Carousel
+          swipeable
+          responsive={responsive}
+          arrows={false}
+          renderDotsOutside
+          showDots
+          dotListClass={classes.dots}
+        >
+          {carouselItems.map((ci) => (
+            <CarouselItem items={ci} />
+          ))}
```

Comment:
Need a `key` here but since `ci` is an array, not sure what's the best way of creating `key` from an array.